tests/tensorflow/r2.12/common.libsonnet

```diff
@@ -74,7 +127,7 @@ local mixins = import 'templates/mixins.libsonnet';
   },
   // Override default schedule for Functional.
   RunNightly:: {
-    schedule: '0 5 * * *',
+    schedule: functional_schedule,
```

Comment:
ditto

Reply:
I reverted and repushed without these changes